### ISSUE TITLE
colBind-kernel for CSRMatrix

### ISSUE THREAD

**Issue by daphne-eu** [daphne-eu/daphne#218]
In GitLab by @pdamme on Mar 11, 2022, 12:50

The `colBind`-operation concatenates two matrices or frames `lhs` and `rhs` horizontally and returns the result as a new data object `res`. The two inputs must have the same number of rows.

**Example**

```
lhs: 1, 2,
     3, 4,

rhs: 0, 1, 0,
     1, 0, 1,

res: 1, 2, 0, 1, 0,
     3, 4, 1, 0, 1,
```

We already have specializations of the `rowBind`-kernel for `DenseMatrix` and `Frame`, but still lack one for `CSRMatrix`.

**This issue asks you to implement**

1. A partial specialization of the kernel struct for `CSRMatrix` as the data type implementation of all input and output matrices.
4. Ideally some test cases.

**Hints**
- Have a look at the [guidelines on implementing a built-in kernel](https://gitlab.know-center.tugraz.at/daphne/prototype/-/blob/master/doc/ImplementBuiltinKernel.md).

**Comment by akroviakov** [comment]
Assigned to @akroviakov.

**Comment by pdamme** [comment]
This issue has just been closed by b2acaea517705f92c70c3b776160c1b634163dfe.
